staging/src/k8s.io/sample-apiserver/pkg/admission/plugin/banflunder/admission.go

```diff
@@ -52,7 +53,17 @@ func (d *disallowFlunder) Admit(a admission.Attributes) error {
 		return nil
 	}
 
+	// on partial update the meta field might not be provided
+	// thus we try to get a name from attributes first
 	flunderName := a.GetName()
+	if len(flunderName) <= 0 {
```

Comment:
yeah, I am with you on that. @liggitt was strongly holding to the fact that  quote "as an example, it should illustrate how you would deal with both types of operations"